### ISSUE TITLE
Querying by multiple map entries using object query notation.

### ISSUE THREAD

**Issue by zippunov** [masumsoft/express-cassandra#124]
Hello. This is simplified example of my table.
```
CREATE TABLE msgs (
  group text,
  created timestamp,
  msg_id text,
  msg_version: text,
  msg_payload: text,
  msg_props map<text, text>,
  PRIMARY KEY(group, created)
)
WITH CLUSTERING ORDER BY (created DESC);

CREATE INDEX ON msgs (msg_id);
CREATE INDEX ON msgs (ENTRIES(msg_props));
```
I want to query this table using several `msg_props` entries.
Like `SELECT *  FROM msgs WHERE group = '...' AND created > '...' AND msg_props['paramA'] = 'value1' AND msg_props['paramB'] = 'value2' ALLOW FILTERING;`
Is there way to build query using object query notation of express-cassandra?

**Comment by masumsoft** [comment]
You could ideally use something like the following:

```
models.instance.Msgs.find({
    group: '...',
    created: { $gt: new Date('...') },
    msg_props: {$contains: {paramA: 'value1', paramB: 'value2'}}
}, { raw: true, allow_filtering: true }, function(err, msgs){

});
```

But there is an issue that prevents you from doing that. The issue has already been fixed and will be released with v2.0.0 soon.

**Comment by masumsoft** [comment]
The issue is now fixed in v2.0.0
